### PR TITLE
Fixes lookup for HELM Monomer 'D'

### PR DESCRIPTION
**Pull request by bp-kelley**
#### What does this implement/fix? Explain your changes.

Simple fix for looking up Helm Monomer 'D'

### CODE REVIEW

File: Code/GraphMol/FileParsers/SequenceParsers.cpp [hunk 1]
```diff
@@ -1187,6 +1187,7 @@ static const char *LookupHELMPeptideMonomer(const char *ptr) {
       break;
     case 'D':
       if (ptr[1] == '\0') return "ASP";
+      break;
```
Comment (greglandrum) [inline Code/GraphMol/FileParsers/SequenceParsers.cpp]: Though it's clearly stylistically correct, I don't think this actually does anything.

Comment (bp-kelley) [inline Code/GraphMol/FileParsers/SequenceParsers.cpp]: It does if ptr[1] is not 0

Comment (greglandrum) [inline Code/GraphMol/FileParsers/SequenceParsers.cpp]: If ptr[1] is not 0, it falls through to the next case statement, sees that ptr[1] is not 0, and then breaks.

The change is clearly the correct thing to do, but I don't see how it actually fixes anything.

Comment (bp-kelley) [inline Code/GraphMol/FileParsers/SequenceParsers.cpp]: This came from roger by the way...

Comment (bp-kelley) [inline Code/GraphMol/FileParsers/SequenceParsers.cpp]: I agree it isn't a bug and is more cosmetic.

**Pull request merged**